cellarium/ml/models/scvi.py

```diff
@@ -445,7 +433,19 @@ def __init__(
     def reset_parameters(self) -> None:
         for m in self.modules():
             m.apply(weights_init)
-        torch.nn.init.normal_(self.px_r, mean=0.0, std=1.0)
+
+        if self.dispersion == "gene":
+            torch.nn.init.normal_(self.px_r, mean=0.0, std=1.0)
+        elif self.dispersion == "gene-label":
+            raise NotImplementedError
+        elif self.dispersion == "gene-cell":
+            torch.nn.init.zeros_(self.px_r)
+        else:
+            raise ValueError(
+                "dispersion must be one of ['gene', "
+                " 'gene-label', 'gene-cell'], but input was "
+                "{}".format(self.dispersion)
+            )
```

Comment:
I think we want to keep this stuff in `__init__()`.

`self.px_r` is only used if `self.dispersion == "gene"` or if `self.dispersion == "gene_label"`, but `self.px_r` is a different size in the two cases.  We don't want to be creating the parameter in `reset_parameters()`, we need to create parameters in `__init__()`.  So we have to have this if-statement in `__init__()` anyway (if we ever want "gene-label" to work).  I think it's easier to just have this here:
```python
torch.nn.init.normal_(self.px_r, mean=0.0, std=1.0)
```